functional/iak-idevid-register-with-certificates/test.sh

```diff
@@ -0,0 +1,157 @@
+#!/bin/bash
+# vim: dict+=/usr/share/beakerlib/dictionary.vim cpt=.,w,b,u,t,i,k
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+HTTP_SERVER_PORT=8080
```

Comment:
I believe you forgot to remove this line, it comes from the revocation webhook scenario.

Reply:
This line is removed now